src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMeanIterative.h

```diff
@@ -39,6 +39,7 @@
 #include <OpenMS/CONCEPT/Exception.h>
 #include <OpenMS/DATASTRUCTURES/ListUtils.h>
 #include <vector>
+#include <algorithm>
```

Comment:
in other files the specific reason for a (non-obvious) include was commented. E.g. `#include <algorithm> // for std::max_element`

Reply:
thanks for your commend :)